### PR TITLE
cmake: add cmake config file to enable importing into duroslight

### PR DESCRIPTION
**Pull request by anton-lb**
Added cmake/nvml-config.cmake

Issue: LBM1-7511

Signed-off-by: Anton Eidelman <anton@lightbitslabs.com>

### CODE REVIEW

**Comment by anton-lb** [comment]
@oren-lb are you ok with this change on master? It does not affect the build, so should be 0 risk.

File: cmake/nvml-config.cmake [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "libpmem.h"
+  PATHS "$ENV{WORKSPACE_TOP}/nvml"
+  PATH_SUFFIXES "src/include"
+)
+
+find_library(Pmem-sdk_LIBRARY NAMES libpmem.a PATHS "${PMEM-SDK_BUILD}/lib")
+
+lb_version(PMEM-SDK_VERSION nvml pmem-sdk)
+
+find_package_handle_standard_args(Pmem-sdk DEFAULT_MSG
```
Comment (maor-lb) [inline cmake/nvml-config.cmake:16]: seems OK but any reason not to use pmem shipped in ubuntu 18.04 ?

Comment (anton-lb) [reply cmake/nvml-config.cmake:16]: @maor-lb Good point, I'll check with libpmem-dev.
Still I'd merge this so our nvml would be available.

**Comment by anton-lb** [comment]
Obsolete. We're using the ubuntu libpmem-dev.

**Pull request closed without merging**